package.json

```diff
@@ -16,7 +16,10 @@
   "dependencies": {
     "@rails/webpacker": "^4.0.7",
     "list.js": "1.1.1",
-    "mirador": "^3.0.0-alpha.9"
+    "mirador": "^3.0.0-alpha.9",
+    "mirador-share-plugin": "^0.1.0",
+    "react": "^16.8.6",
```

Comment:
We have to add `react` and `react-dom` dependencies now that we are using the `es` version of the package. The `umd` has all of this baked into it (which we don't want).